### DISCUSSION TITLE
Copilot stopped working; it says that subscription has expired, I can't log in again

### DISCUSSION THREAD

**Discussion by forest-spells** [community/community#45602] [Copilot Conversations] [answered]
I use PyCharm.
I have access to Copilot through the university.

A month ago, I got a message that the subscription had expired. I was logged out of GitHub, so I tried to log in again through PyCharm; the message "you are all set in" appears in the browser. But PyCharm still shows that I'm not logged in to GitHub, so I can't use Copilot.

I've tried installing Copilot again, and also updating the plugin.

**Comment by jge162** [comment, marked as answer]
@Wiktoria999 When you are prompted to login into copilot does it take you to GitHub website for you to authorize its use in PyCharm?
try clearing your browser cash, also try reseting copilot in settings(of Pycharm). this should fix issue.

**Reply by forest-spells** [reply]
Yes, it takes me to the GitHub page, I rewrite the authorization code, I get a success message. Unfortunately, in PyCharm I am still not logged in.
I tried clearing the cache in the browser and resetting Copilot, but that didn't help.

**Reply by jge162** [reply]
It sounds like the issue may be with the integration of GitHub with PyCharm. One potential solution would be to try re-connecting your GitHub account with PyCharm by going to the settings or preferences menu in PyCharm and navigating to the "Version Control" or "Integrations" section. Make sure that the GitHub option is selected and that you have entered the correct login credentials. Or maybe log out and back in again too.

**Reply by jge162** [reply]
<img width="534" alt="Screenshot 2023-01-28 at 1 01 39 AM" src="https://user-images.githubusercontent.com/31228460/215257260-8bc1bc54-5335-4b1b-beb5-a3d392dcf770.png">

**Reply by forest-spells** [reply]
The problem is over. I went into the settings of my GitHub account and found that Copilot was disabled. I don't know how, because I didn't do anything there. Anyway, I activated Copilot there and now it works.
